res/css/structures/_SpacePanel.scss

```diff
@@ -133,8 +146,8 @@ $activeBorderColor: $secondary-fg-color;
         }
 
         .mx_SpaceButton_toggleCollapse {
-            width: calc($gutterSize - $activeStripeSize);
-            margin-left: 1px;
+            width: $gutterSize;
+            margin-right: -4px;
```

Comment:
any time we do negative margins it's not a good sign. Can we get a comment to explain what is going on here?